src/Resources/templates/htmlTableOutput.phtml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SensioLabs DeprecationDetector</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
Looking at the markup, I'd say that the report should display fine offline, just the bootstrap style will be missing.

Inlining 120 KB bootstrap code probably isn't they way to go either. Maybe we should replace it with a few lines of good old handcrafted CSS? :smiley: